oml/functional/knn.py

```diff
@@ -53,7 +54,7 @@ def batched_knn(
     gt_ids = []
 
     # we do batching over first (queries) dimension
-    for i in range(0, nq, bs):
+    for i in tqdm(range(0, nq, bs), desc="Finding nearest neighbors."):
```

Comment:
verbose?

Reply:
Having `verbose` parameter means adding this parameter to the all chain on function calls and to the config.
I don't think having this print may hurt... So, let's just keep it